src/domains/pay-wallet-app/api/payment-wallet/v1/_openapi.json.tpl

```diff
@@ -422,6 +478,16 @@
               }
             }
           },
+          "401": {
+            "description": "Unauthorized: the provided token is not valid or expired.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ProblemJson"
+                }
+              }
+            }
```

Comment:
remove also this body in order to be aligned with other api's

Reply:
Fixed